src/components/__tests__/HierarchicalMenu.js

```diff
@@ -113,7 +113,10 @@ const defaultState = {
     microsoft,
   ],
   refine: () => {},
-  createURL: () => {},
+  createURL: () => {}, // TODO: turn this into a string
```

Comment:
Is that still to do?

Reply:
good catch, not necessary

```suggestion
  createURL: () => {},
```